playbooks/roles/appserver/defaults/main.yml

```diff
@@ -1,18 +1,14 @@
 ---
 appserver_instance_id: null
 
-# The local ports that will be tunnelled to memcached and redis-sentinel.
+# The local ports that will be tunnelled to memcached and redis.
 appserver_memcached_ports:
   - 11211
   - 11212
   - 11213
-appserver_redis_sentinel_ports:
-  - 6379
-  - 6380
-  - 6381
+appserver_redis_port: 6379
 
 # Build upstream specs for the Consul Connect command line.
 appserver_memcached_upstream_specs: |-
   {% for port in appserver_memcached_ports %} -upstream memcached-{{ appserver_instance_id }}-{{ loop.index0 }}:{{ port }}{% endfor %}
-appserver_redis_sentinel_upstream_specs: |-
-  {% for port in appserver_redis_sentinel_ports %} -upstream redis-sentinel-{{ appserver_instance_id }}-{{ loop.index0 }}:{{ port }}{% endfor %}
+appserver_redis_upstream_specs: " -upstream redis-{{ appserver_instance_id }}-:{{ appserver_redis_port }}"
```

Comment:
Nit: There's a `-` before the colon and port. (In fact this is probably not a nit, as it may simply not work otherwise).

Reply:
I removed the job index, but failed to remove the dash.  Definitely not a nit! :)